File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGPositionType.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGPositionType {
    YGPositionTypeStatic,
    YGPositionTypeRelative,
    YGPositionTypeAbsolute;

    public static YGPositionType forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGPositionType.kt
package app.cash.redwood.yoga.internal.enums

enum class YGPositionType {
  YGPositionTypeStatic,
  YGPositionTypeRelative,
  YGPositionTypeAbsolute;

  fun getValue(): Int {
    return ordinal
  }

  companion object {
    fun forValue(value: Int): YGPositionType {
      return values()[value]
    }
  }
}
